### PR TITLE
Multiple translations, also retrying to get issues marked as complete.

### PR DESCRIPTION
**Pull request by frenchiveruti**
Applying for issue(s):
- #1675
- #1755
- #1773
- #1781
- #1789
- #1795
- #1809
- #1821
- #1826
- #1858
- #1876
- #1952
- #1960
- #1966
- #1993
- #2005
- #2121
- #2133
- #2140
- #2181
- #2188

### CODE REVIEW

**Comment by frenchiveruti** [comment]
A thing to notice is that several issues listed here were already fixed (by me) and they were not counted.  
(see for example: https://github.com/OpenRCT2/Localisation/pull/1836)

File: data/language/es-ES.txt [hunk 1]
```diff
@@ -3727,9 +3763,9 @@ STR_PARK    :Torres de marfil
 STR_DTLS    :Un parque bien establecido, con algunos problemas.
 
 <Rainbow Valley>
-STR_SCNR    :Valle del Arco Iris
-STR_PARK    :Valle del Arco Iris
-STR_DTLS    :Las autoridades del Valle del Arco Iris no te dejarán introducir cambios paisajísticos ni eliminar árboles grandes; aún así, debes convertir el área en un gran parque temático.
+STR_SCNR    :Valle del Arcoíris
+STR_PARK    :Valle del ArcoíIris
```
Comment (Gymnasiast) [inline data/language/es-ES.txt:3767]:
```suggestion
STR_PARK    :Valle del Arcoíris
```

Comment (francisco-lafe) [reply data/language/es-ES.txt:3767]: Hi Michael.
I don't know how to accept your change but the change is correct,

Thanks,

Comment (tupaschoal) [reply data/language/es-ES.txt]: I've done it for you, but it was just a matter of clicking this "Commit suggestion" button (if you're on desktop) :)
image

Comment (frenchiveruti) [reply data/language/es-ES.txt]: Lol, thanks.

**Pull request merged**